### PR TITLE
Verifying user and group in lnms

### PR DESCRIPTION
**Pull request by PJGuyTen**
Update lnms to verify current running user and group

### CODE REVIEW

**Comment by PJGuyTen** [comment]
> This works, but it might be better to fix this in yasu

I'm unfamiliar with yasu.  If you can point me to the open source project and I have the expertise to fix it, I will; however, I'd rather merge this fix while I look into yasu instead of leaving it uncorrected even temporarily.  I can always revert it after a fix for yasu is made.

**Comment by PJGuyTen** [comment]
> > This works, but it might be better to fix this in yasu
> 
> I'm unfamiliar with yasu. If you can point me to the open source project and I have the expertise to fix it, I will; however, I'd rather merge this fix while I look into yasu instead of leaving it uncorrected even temporarily. I can always revert it after a fix for yasu is made.

https://github.com/crazy-max/yasu

**Comment by crazy-max** [comment]
@PJGuyTen What is the issue with the current logic?

**Comment by murrant** [comment]
@crazy-max the issue is it fails if you run it as the librenms user.  For example try running `lnms scan`

File: rootfs/usr/bin/lnms [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
+if [[ "$currUser" == "$LIBRENMS_USER" ]]; then
+  if [[ "$currGroup" == "$LIBRENMS_GROUP" ]]; then
+    php -f /opt/librenms/lnms "$@"
+  else
+    yasu librenms:librenms php -f /opt/librenms/lnms "$@";
+  fi
+else
+  yasu librenms:librenms php -f /opt/librenms/lnms "$@";
+fi
```
Comment (crazy-max) [inline rootfs/usr/bin/lnms:14]: Should be enough:

```suggestion
if [ "$(id -u -n)" = "librenms" -a "$(id -g -n)" = "librenms" ]; then
  php -f /opt/librenms/lnms "$@"
else
  yasu librenms:librenms php -f /opt/librenms/lnms "$@"
fi
```

Comment (PJGuyTen) [reply rootfs/usr/bin/lnms:14]: @crazy-max `id` isn't available on all distributions, which is why I used `ps`

Comment (crazy-max) [reply rootfs/usr/bin/lnms:14]: This script is not intended to be used outside the container. What's your use case?

Comment (PJGuyTen) [reply rootfs/usr/bin/lnms:14]: @crazy-max exactly as you state; however, I didn't want to limit you or anyone else in case the container changed.

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Pull request merged**